examples/gan/gan_mnist_pl/gan.py

```diff
@@ -113,7 +113,8 @@ def __init__(
     def forward(self, z):
         return self.generator(z)
 
-    def adversarial_loss(self, y_hat, y):
+    @staticmethod
```

Comment:
it's because the PTL adversarial_loss method API changed, and if we don't conform to it, I get an override check error about the parent not defining the method.